src/System.Private.CoreLib/shared/System/TimeZoneInfo.AdjustmentRule.cs

```diff
@@ -100,6 +102,7 @@ public sealed class AdjustmentRule : IEquatable<AdjustmentRule>, ISerializable,
                 TimeSpan baseUtcOffsetDelta,
                 bool noDaylightTransitions)
             {
+                AdjustDaylightDeltaToExpectedRange(ref daylightDelta, ref baseUtcOffsetDelta);
```

Comment:
This is method call which not sure if it will be inlined. why you don't have the method code move here. it is not big anyway.

Reply:
@tarekgh if perf becomes a problem here, I'll fix it - I'd rather make sure coreclr can optimize paths like this (method called from exactly one place)